### PR TITLE
Add support for DateRange types

### PR DESCRIPTION
**Pull request by WickyNilliams**
### What is the context of this PR?
Adds support for DateRange answer type

### How to review 
Run tests, review code, preview through runner and ensure DateRange handled correctly

### CODE REVIEW

**Comment by WickyNilliams** [comment]
[Date range selection answer type (3)](https://trello.com/c/NbysNHRa/74-date-range-selection-answer-type-3)

File: src/components/Answers/Dummy/DateRange/index.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+import DummyTextInput from "../TextInput";
+import { colors } from "constants/theme";
+import chevronIcon from "components/Accordion/chevron.svg";
+
+const GUTTER = "0.625em";
+
+const Field = styled.div`
+  display: inline-block;
+  width: calc(${props => props.width} - ${GUTTER});
+  margin-left: ${GUTTER};
```
Comment (hamishtaplin) [inline src/components/Answers/Dummy/DateRange/index.js]: use `:not(:first-of-type)` to negate need for negative margin on container maybe?

File: src/components/Answers/Dummy/DateRange/index.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  margin-bottom: 0.875rem;
+`;
+
+const DatePickerWrapper = styled.div`
+  margin-left: -${GUTTER};
+`;
+
+export const DatePicker = () => (
+  <DatePickerWrapper>
+    <Field width="25%">
```
Comment (hamishtaplin) [inline src/components/Answers/Dummy/DateRange/index.js]: Is the `width` prop supposed to end up on the rendered `div`?

screenshot 2017-10-27 12 26 45

File: src/components/Answers/Dummy/DateRange/index.js (hunk 1 above)
Comment (WickyNilliams) [reply src/components/Answers/Dummy/DateRange/index.js]: Used `&:first-of-type { margin-left: 0; }`

**Pull request merged**